tests/TESTS/LinkLoss/device/source/DisconnectionService.h

```diff
@@ -0,0 +1,61 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef DISCONNECTION_SERVICE_H
+#define DISCONNECTION_SERVICE_H
+
+#include "ble/BLE.h"
+
+#if BLE_FEATURE_GATT_SERVER
+
+#include "ble/Gap.h"
+#include "ble/gap/ChainableGapEventHandler.h"
+
```

Comment:
Please add a short comment block explaining what this is and how it works.

My advice is to not "resolve conversion" on github until you actually push fixes, otherwise problems will slip through even if someone catches them once,